src/pages/prescreen-1a.tsx

```diff
@@ -55,13 +58,15 @@ export default class Prescreen1c extends React.Component {
 
   handleDateChange(name, value){
     if (name === "birthDatePicked") {
+      console.log('this')
```

Comment:
```suggestion
```

Reply:
They were for debugging, forgot to remove em, will do so and check for others.